hw/ip_templates/alert_handler/doc/checklist.md

```diff
@@ -131,21 +131,21 @@ Testbench     | [TB_TOP_CREATED][]                    | Done        |
 Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
 Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
 Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Done        |
-Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | waived      | Revisit later. Tool setup in progress.
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
 Testbench     | [TB_GEN_AUTOMATED][]                  | N/A         |
 Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
 Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Done        |
-Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | waived      | Revisit in V2S.
+Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | N/A         |
```

Comment:
does that mean we are only signing off based on simulation coverage, and do not apply this to any of the FPV testbenches?

Reply:
Yes. That is what I believed.
In V2 we excluded prim_alert_sender and prim_esc_receiver from the coverage.
They are covered in FPV and DV (non-VCS) testbenchs. And they all passed the V2 requirements.
I think here the `FPV` testbenchs we are referring to the sec_cm bench (where we black box cm related modules and check if design will trigger internal alerts).

If what I understand correctly, we will do `N/A` here and mark the `V2S` fpv section to done.

Reply:
Ok that SGTM.